sources/api/migration/migrations/v1.16.0/kernel-modules-autoload-restart/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use migration_helpers::common_migrations::{ListReplacement, ReplaceListsMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added a new `autoload` setting to `settings.kernel.modules`, which needs
+/// re restart of `systemd-modules-load.services`.
+fn run() -> Result<()> {
+    migrate(ReplaceListsMigration(vec![ListReplacement {
+        setting: "services.kernel-modules.restart-commands",
+        old_vals: &[],
+        new_vals: &["/usr/bin/systemctl restart systemd-modules-load"],
```

Comment:
`try-restart` is safer since it won't cause services to be queued for starting unless they're already started, which could create issues during boot. It's not specifically a problem here since `systemd-modules-load.service` starts very early, but I'd prefer to stick to the "safer" usage everywhere.

```suggestion
        new_vals: &["/usr/bin/systemctl try-restart systemd-modules-load"],
```

Reply:
systemd activation states and lingo around these is somewhat confusing or overloaded.

The man page says `try-restart` is only doing its work when the service is running (`This does nothing if units are not running.`). The activation state for `systemd-modules-load` is `active (exited)` once it ran (in contrast to `active (running)`).

I confirmed that it is still considered what is called running in the manpage by manually triggering a `systemctl try-reload systemd-modules-load` and checking the time the service last ran. Added the changes accordingly.